integration-tests/src/test/java/uk/gov/di/authentication/api/AuthorisationIntegrationTest.java

```diff
@@ -109,7 +109,7 @@ class AuthorisationIntegrationTest extends ApiGatewayHandlerIntegrationTest {
     private static final String CLAIMS =
             "{\"userinfo\":{\"https://vocab.account.gov.uk/v1/coreIdentityJWT\":{\"essential\":true},\"https://vocab.account.gov.uk/v1/address\":null}}";
 
-    private static IntegrationTestConfigurationService configuration =
+    private static final IntegrationTestConfigurationService configuration =
```

Comment:
You're going to get a conflict with https://github.com/govuk-one-login/authentication-api/pull/3911 on this. This is a good reason not to make unrelated changes in other files 🙃

Reply:
This was meant to be in a different PR, will move